Mathlib/GroupTheory/GroupExtension/Defs.lean

```diff
@@ -143,13 +143,49 @@ theorem inl_conjAct_comm {e : E} {n : N} : S.inl (S.conjAct e n) = e * S.inl n *
   simp only [conjAct, MonoidHom.coe_mk, OneHom.coe_mk, MulEquiv.trans_apply,
     MonoidHom.apply_ofInjective_symm, MulAut.conjNormal_apply, MonoidHom.ofInjective_apply]
 
-/-- `GroupExtension`s are equivalent iff there is a homomorphism making a commuting diagram. -/
+/-- `GroupExtension`s are equivalent iff there is an isomorphism making a commuting diagram. -/
```

Comment:
Can you point to `Equiv.ofMonoidHom` (and include the file name) in the docstring?